omnibus/files/private-chef-cookbooks/private-chef/templates/default/pedant_config.rb.erb

```diff
@@ -75,7 +75,7 @@ search_server "<%= @solr_url %>"
 
 <% if node['private_chef']['opscode-erchef']['search_provider'] == "elasticsearch" %>
 search_commit_url "/_refresh"
-search_url_fmt "/chef/_search?fq=X_CHEF_type_CHEF_X:%{type}&q=%{query}"
+search_url_fmt "/chef/_search?q=X_CHEF_type_CHEF_X:%{type}%%20%{query}"
```

Comment:
Elastic search experts: Can you confirm if this is a legit change.  This is what @lancewf and I came up with to get the current tests passing but maybe there's an edge case we're missing.